core/src/main/java/hudson/model/Queue.java

```diff
@@ -866,7 +841,7 @@ private WaitingItem peek() {
      * @since 1.483
      */
     public List<Item> getApproximateItemsQuickly() {
```

Comment:
Should this (and the corresponding method in `View`) not be deprecated and the calls from `BuildHistoryWidget` and `ComputerSet/sidepanel.jelly` (and documentation in `queue.jelly`) updated?